app/models/journeys/get_a_teacher_relocation_payment/answers_presenter.rb

```diff
@@ -14,6 +14,10 @@ def eligibility_answers
           end
           a << start_date_details
           a << subject_details
+          # FIXME RL we'll soon be removing trainee screens from the journey
+          # so we're just adding this for the time being to save having to
+          # write a test we'll be deleting in the next commit
+          a << visa_details unless answers.trainee?
```

Comment:
See this pr https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/2905